db_upgrade_scripts/mosip_regprc/sql/1.1.5.5_to_1.2.0.1-B1_rollback.sql

```diff
@@ -0,0 +1,8 @@
+\c mosip_regprc sysadmin
+
+ALTER TABLE regprc.registration DROP COLUMN IF EXISTS resume_timestamp;
+ALTER TABLE regprc.registration DROP COLUMN IF EXISTS default_resume_action;
```

Comment:
@ckm007  As part of release-1.2.0.1 many other changes for same table and other tables in regproc, Why it is added only for these two columns here